docs/source/faq.rst

```diff
@@ -5,6 +5,64 @@ This page contains answers to several frequently asked technical questions
 about Calico. It is updated on a regular basis: please check back for more
 information.
 
+What is Calico?
+---------------
+
+See also http://www.projectcalico.org/why-calico/
+The problem Calico tries to solve is the networking of workloads (VMs,
+containers, etc) in a high scale environment.  Existing L2 based methods for
+solving this problem have problems at high scale.  Compared to these, we think
+Calico is more scalable, simpler and more flexible.  We think you should look
+into it if you have more than a handful of nodes on a single site.
+
+Does Calico work with IPv6?
+---------------------------
+
+Yes!  We have demonstrated IPv6 with Calico on Openstack and Docker/Powerstrip.
+
+Why in Calico do we configure Neutron/ML2 to use 'local' and 'flat' networks?
+-----------------------------------------------------------------------------
```

Comment:
Given Cory's view, I recommend taking this out for now, so that we can merge the rest.